language/move-prover/tests/sources/restrictions.move

```diff
@@ -0,0 +1,60 @@
+// no-boogie-test
```

Comment:
Is this comment interpreted by the testing tool?

Reply:
Yes